src/modules/personenkontext/api/find-rollen.response.ts

```diff
@@ -2,9 +2,14 @@ import { ApiProperty } from '@nestjs/swagger';
 import { Rolle } from '../../rolle/domain/rolle.js';
 
 export class FindRollenResponse {
+    public constructor(moeglicheRollen: Rolle<true>[], total: number) {
+        this.moeglicheRollen = moeglicheRollen;
+        this.total = total;
+    }
+
     @ApiProperty({ type: [Rolle<true>] })
-    public moeglicheRollen!: Rolle<true>[];
+    public readonly moeglicheRollen: Rolle<true>[];
```

Comment:
Needs to be a response-object (probably `RolleResponse`) for proper API definitions.

Reply:
adjusted, mapping happening in private method/constructor